04_HowTos/QueryClient/How_to_query_DESI_DR1_Data.ipynb

```diff
@@ -0,0 +1,967 @@
+{
```

Comment:
<p>It might be useful to add a clarification to the bullet "Only quantities derived from&nbsp;<em>cumulative</em>&nbsp;tile-based spectra are included at the present time." because it may give the impression that this applies to the full dataset but there are tables regarding the exposures and the zpix table includes quantities from the full-depth healpix-based spectra. Did you mean for this comment to only apply to the <code>ztile</code> table? I suggest either listing which table(s) this statement concerns or otherwise removing it.  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/astro-datalab/notebooks-latest/pull/264/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='3'/>

Reply:
<p>Good point. Indeed, that only applies to <code>ztile</code>. I'll clarify that.  </p>